devsecops/security-scanning/xebialabs/README.md.tmpl

```diff
@@ -0,0 +1,47 @@
+{{$app := .AppName | kebabcase}}
+## Simple security and compliance template and dashboard
+
+Security and compliance are critical aspects of product release. Organizations struggle to track application release status information and understand security and compliance risks across many different applications, teams, and environments. When risk assessment, security testing, and compliance checks aren’t built into the Continuous Integration/Continuous Delivery (CI/CD) pipeline, releases fail and cause delays, security vulnerabilities threaten production, and IT governance violations result in expensive fines.
+
+This blueprint provides a way to integrate tools like Sonarqube, Fortify, Checkmarx, and Black duck into release pipeline.
+
+#### Sonarqube
+
+SonarQube is an open source platform for continuous inspection of code quality to perform automatic reviews with static analysis of code to detect bugs, code smells, and security vulnerabilities in application source code. 
+
+
+#### Fortify SSC
+
+Fortify Software Security Center (SSC) provides centralised management of their application security testing. Security teams use SSC to review and manage security testing activities, prioritize remediation efforts based on risk potential, measure improvements and generate cross portfolio management reports.
+
+#### Fortify on Demand
+
+Fortify on Demand provides software security analysis as a service. It can evaluate application code against selected security metrics.
+
+#### Checkmarx
+
+Checkmarx provides Checkmarx Static Application Security Testing (CxSAST) and Checkmarx Open Source Analysis (CxOSA). CxSAST automatically scan uncompiled code and identify security vulnerabilities. CxOSA provides a way to control and manage open source components and mitigate potential risks to the application, organizations and its users.
+
+### Black duck
+
+Black duck provides a complete open source management solution. It can identify potential security risks, license risks and operational risks for open source components.
+
+### Prerequisites
+
+If you do not have XL Release instances running, you can use this Docker Compose setup to spin them up. The setup also contains a preconfigured Jenkins instance and a dockerproxy container.
+Please note that you need to obtain a trial license or bring your own license for XL Release and XL Deploy. Follow the steps below to use the Docker Compose files.
+
+    1. Download the contents of https://github.com/xebialabs/e-commerce-microservice/tree/master/docker-compose/xebialabs into a folder such as `xl-platform`.
```

Comment:
@deepu105 it feels like this docker-compose setup is used globally in almost all of the blueprints and it makes sense actually, so how about de-coupling it from the e-commerce microservice sample app and create a separate proper repo/dist folder for it, does it makes sense?

Reply:
in the e-commerce microservice repo we can still include it as a nested-repo (sub-module)..